src/styles/blocks/price.scss

```diff
@@ -0,0 +1,21 @@
+.price {
```

Comment:
is this need to be another bem block?
6'th point in checklist

Reply:
Not fixed